doc/HOWTO_RELEASE.rst.txt

```diff
@@ -352,36 +352,26 @@ Also create a new version hash in cversions.txt and a corresponding version
 define NPY_x_y_API_VERSION in numpyconfig.h
 
 
-Trigger the wheel builds on travis-ci and Appveyor
---------------------------------------------------
-See the `numpy wheels` repository.
+Trigger the wheel builds
+------------------------
+See the `MacPython/numpy wheels` repository.
 
 In that repository edit the files:
 
-- ``.travis.yml``;
-- ``appveyor.yml``.
+- ``azure/posix.yml``
+- ``azure/windows.yml``.
 
 In both cases, set the ``BUILD_COMMIT`` variable to the current release tag -
-e.g. ``v1.11.1``.
+e.g. ``v1.19.0``.
 
 Make sure that the release tag has been pushed.
 
-Trigger a build by doing a commit of your edits to ``.travis.yml`` and
-``appveyor.yml`` to the repository::
-
-    cd /path/to/numpy-wheels
-    # Edit .travis.yml, appveyor.yml
-    git commit
-    git push
-
-The wheels, once built, appear at a Rackspace container pointed at by:
-
-- http://wheels.scipy.org
-- https://3f23b170c54c2533c070-1c8a9b3114517dc5fe17b7c3f8c63a43.ssl.cf2.rackcdn.com
-
-The HTTP address may update first, and you should wait 15 minutes after the
-build finishes before fetching the binaries.
+Trigger a build by pushing a commit of your edits to the repository. Note that
+you can do this on a branch, but it must be pushed to the
+``MacPython/numpy-wheels`` repository before the uploads will happen since only
```

Comment:
Maybe "pushed upstream to the ``MacPython/numpy-wheels`` repository to trigger uploading.

Reply:
ok